packages/hash/frontend/src/blocks/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { paragraphBlockComponentId } from "@hashintel/hash-shared/blocks";
+import { componentNodeToId } from "@hashintel/hash-shared/prosemirror";
+import { ProsemirrorNode, Schema } from "prosemirror-model";
+
+export const isParagraphNode = (node: ProsemirrorNode<Schema>) => {
```

Comment:
This can go in `shared/prosemirror.ts`.

Reply:
Moved it here https://github.com/hashintel/hash/pull/953/commits/1fe559aa3502eac4bd84690e6edf3384a9fee7c1